defaults/manager_versions.yaml

```diff
@@ -2,32 +2,32 @@ manager_repos_by_version:
   "master_latest":
     centos7: 'https://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
     centos8: 'https://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
-    debian10: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
     debian11: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
+    debian12: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
```

Comment:
I hate the way this file is managed, we don't really need to specify specific distro release like that for years now.

Reply:
I think https://github.com/scylladb/scylla-cluster-tests/pull/8210 would do the trick

Reply:
I just changed debian10 to 12 there, if there's a better way now I can remove this.

Reply:
Okay, I can rebase once that PR is merged.

Reply:
Done